### PR TITLE
Docsp 32563 quick reference

### PR DESCRIPTION
**Pull request by jordan-smith721**
Jira - https://jira.mongodb.org/browse/DOCSP-32563
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/entity-framework/docsworker-xlarge/DOCSP-32563-quick-reference/quick-reference/

### CODE REVIEW

File: source/quick-reference.txt [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Command
+     - Syntax
+
+   * - | **Set up DBContext**
+       |
+       | `API Documentation <https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbcontext>`__
```
Comment (mongoKart) [inline source/quick-reference.txt:19]: Q: The other Quick Ref pages also link to Usage Examples and Fundamentals. What's a good way to make note to come back for those here?

File: source/includes/code-examples/quick-reference.cs [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    {
+        Console.WriteLine(p.ToBsonDocument());
+    }
+    // end-find-many
+}
+
+void InsertOne()
+{
+    // start-insert-one
+    db.Planets.Add(new Planet() { name = "Pluto", hasRings = false, orderFromSun = 9 });
```
Comment (mongoKart) [inline source/includes/code-examples/quick-reference.cs:42]: Bets on how long until someone complains about Pluto?

File: source/quick-reference.txt [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+       |
+       | `API Documentation <https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbcontext.removerange>`__
+
+     - .. literalinclude:: /includes/code-examples/quick-reference.cs
+          :start-after: // start-delete-many
+          :end-before: // end-delete-many
+          :language: csharp
+          :dedent:
+
+   * - | **Set the Order in which to Retrieve Documents**
```
Comment (mongoKart) [inline source/quick-reference.txt:135]: I: "which" is a relative pronoun here so should be capitalized

```suggestion
   * - | **Set the Order in Which to Retrieve Documents**
```

File: source/quick-reference.txt [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+          .. output::
+             :language: json
+             :visible: false
+
+             { "_id" : ObjectId("..."), "name" : "Saturn", "orderFromSun" : 6, "hasRings" : true }
+             { "_id" : ObjectId("..."), "name" : "Uranus", "orderFromSun" : 7, "hasRings" : true }
+             { "_id" : ObjectId("..."), "name" : "Neptune", "orderFromSun" : 8, "hasRings" : true }
+
+   * - | **Project Document Fields When Retrieving Them**
```
Comment (mongoKart) [inline source/quick-reference.txt:241]: S: seems cleaner to leave off "them"

```suggestion
   * - | **Project Document Fields When Retrieving**
```

**Review by mongoKart — APPROVED** [review]
LGTM + a few suggestions

File: source/quick-reference.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/quick-reference.txt:19]: I'll make a ticket to come back and update this page after those are done

**Comment by damieng** [comment]
I think we should change "Setup a DBContext" to "Configuring a DbContext".

Then introduce a section right after that titled "Creating an instance of the DbContext" with this code:

```csharp
var mongoConnectionString = Environment.GetEnvironmentVariable("MONGODB_URI");
var mongoClient = new MongoClient(mongoConnectionString);
var db = PlanetDbContext.Create(mongoClient.GetDatabase("planets"));
```

**Pull request merged**